[2021-07-14 18:17:57,572] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: bdt_2021_ETL.store_opt_2 2021-07-14T18:16:59.219967+00:00 [queued]>
[2021-07-14 18:17:57,602] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: bdt_2021_ETL.store_opt_2 2021-07-14T18:16:59.219967+00:00 [queued]>
[2021-07-14 18:17:57,604] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-14 18:17:57,604] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2021-07-14 18:17:57,606] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-14 18:17:57,673] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): store_opt_2> on 2021-07-14T18:16:59.219967+00:00
[2021-07-14 18:17:57,681] {standard_task_runner.py:52} INFO - Started process 888 to run task
[2021-07-14 18:17:57,692] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'bdt_2021_ETL', 'store_opt_2', '2021-07-14T18:16:59.219967+00:00', '--job-id', '28', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/second_dag.py', '--cfg-path', '/tmp/tmpy274wttc', '--error-file', '/tmp/tmp_4ky5v4i']
[2021-07-14 18:17:57,699] {standard_task_runner.py:77} INFO - Job 28: Subtask store_opt_2
[2021-07-14 18:17:57,945] {logging_mixin.py:104} INFO - Running <TaskInstance: bdt_2021_ETL.store_opt_2 2021-07-14T18:16:59.219967+00:00 [running]> on host bf600838264e
[2021-07-14 18:17:58,250] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=bdt_2021_ETL
AIRFLOW_CTX_TASK_ID=store_opt_2
AIRFLOW_CTX_EXECUTION_DATE=2021-07-14T18:16:59.219967+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-14T18:16:59.219967+00:00
[2021-07-14 18:17:58,293] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2021-07-14 18:17:58,296] {base.py:78} INFO - Using connection to: id: mysql_test_conn. Host: 172.17.0.1, Port: 3310, Schema: project_bdt, Login: root, Password: ***, extra: {}
[2021-07-14 18:17:59,774] {dbapi.py:309} INFO - Loaded 1000 rows into rfam16 so far
[2021-07-14 18:18:01,365] {dbapi.py:309} INFO - Loaded 2000 rows into rfam16 so far
[2021-07-14 18:18:03,313] {dbapi.py:309} INFO - Loaded 3000 rows into rfam16 so far
[2021-07-14 18:18:04,649] {dbapi.py:309} INFO - Loaded 4000 rows into rfam16 so far
[2021-07-14 18:18:06,361] {dbapi.py:309} INFO - Loaded 5000 rows into rfam16 so far
[2021-07-14 18:18:08,526] {dbapi.py:309} INFO - Loaded 6000 rows into rfam16 so far
[2021-07-14 18:18:09,882] {dbapi.py:309} INFO - Loaded 7000 rows into rfam16 so far
[2021-07-14 18:18:10,583] {dbapi.py:312} INFO - Done loading. Loaded a total of 7420 rows
[2021-07-14 18:18:10,584] {python.py:151} INFO - Done. Returned value was: None
[2021-07-14 18:18:10,636] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=bdt_2021_ETL, task_id=store_opt_2, execution_date=20210714T181659, start_date=20210714T181757, end_date=20210714T181810
[2021-07-14 18:18:10,770] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-07-14 18:18:10,806] {local_task_job.py:151} INFO - Task exited with return code 0
